knora/excel2xml.py

```diff
@@ -229,72 +217,28 @@ def find_date_in_string(string: str, calling_resource: str = "") -> Optional[str
         return None
 
 
-def _check_and_prepare_values(
-    value: Optional[Union[PropertyElement, str, int, float, bool]],
-    values: Optional[Iterable[Union[PropertyElement, str, int, float, bool]]],
-    name: str,
-    calling_resource: str = ""
+def prepare_value(
+    value: Union[PropertyElement, str, int, float, bool, Iterable[Union[PropertyElement, str, int, float, bool]]]
 ) -> list[PropertyElement]:
     """
-    There is a variety of possibilities how to call a make_*_prop() method. Before such a method can do its job, the
-    parameters need to be checked and prepared, which is done by this helper method. The parameters "value" and "values"
-    are passed to it as they were received. This method will then perform the following checks, and throw a BaseError in
-    case of failure:
-      - check that exactly one of them contains data, but not both.
-      - check that the values are usable, and not N/A
-
-    Then, all values are transformed to PropertyElements and returned as a list. In case of a single "value", the
-    resulting list contains the PropertyElement of this value.
+    This method transforms the parameter "value" from a make_*_prop() method into a list of PropertyElements. "value" is
+    passed on to this method as it was received.
 
     Args:
         value: "value" as received from the caller
-        values: "values" as received from the caller
-        name: name of the property (for better error messages)
-        calling_resource: name of the resource (for better error messages)
 
     Returns:
         a list of PropertyElements
     """
+    # make sure that "value" is list-like
+    if not isinstance(value, Iterable) or isinstance(value, str):
+        value = [value, ]
```

Comment:
```suggestion
        value = [value]
```
the trailing comma is only needed for tuples